drake/systems/plants/RigidBody.h

```diff
@@ -324,6 +320,10 @@ class DRAKERBM_EXPORT RigidBody {
 #endif
 
  private:
+  // TODO(tkoolen): It's very ugly, but parent, dofnum, and pitch also exist
```

Comment:
@RussTedrake wrote that comment I believe, but you can leave my name there if you want.